qlib/data/dataset/processor.py

```diff
@@ -106,6 +106,22 @@ def process_inf(df):
 
         return replace_inf(df)
 
+class Fillna(Processor):
+    """Process infinity  """
+
+    def __call__(self, df):
+        def fill_na(data):
+            def process_na(df):
+                for col in df.columns:
```

Comment:
You don't have to follow the [previous weird style of processor](https://github.com/you-n-g/qlib/blob/main/qlib/data/dataset/processor.py#L99)